### PR TITLE
feat: add aria attributes for MiniToc

### PR DESCRIPTION
**Pull request by Pavel-Tyan**
add aria-label to MiniToc and aria-current for MiniToc headings.

Task: https://github.com/orgs/diplodoc-platform/projects/2/views/1?pane=issue&itemId=55495812

### CODE REVIEW

**Comment by martyanovandrey** [comment]
Left some comments here 
https://github.com/diplodoc-platform/components/issues/217#issuecomment-2210561173

- aria-current already added for `<li>` elements
- aria-label should be used with translation like `aria-label={t('description')}`in [this file](https://github.com/diplodoc-platform/components/blob/6bf0c0b45aea3d76bb952ae1b46da5e11915e70c/src/components/MiniToc/MiniToc.tsx#L84C17-L84C46)
translation can be added only for russian and english

File: src/components/MiniToc/MiniToc.tsx [hunk 1]
```diff
@@ -47,8 +47,32 @@ const MiniToc = memo<MinitocProps>(({headings, router, headerHeight, onItemClick
...
+                <a href={href} className={b('section-link')} data-router-shallow>
+                    {title}
+                </a>
+            </li>
+        );
+    });
+
     return (
-        <div className={b()}>
+        <div className={b()} aria-label={t('article navigation')}>
```
Comment (martyanovandrey) [inline src/components/MiniToc/MiniToc.tsx:75]: label should be `article-navigation` and it need to be aded here and in "ru" 
https://github.com/diplodoc-platform/components/blob/master/src/i18n/en.json#L50

**Review by Pavel-Tyan — COMMENTED** [review]
Before using nav and h1-h6 tags:

https://github.com/diplodoc-platform/components/assets/87060818/12d46c82-abdd-460e-8b1a-574651498679

After: 

https://github.com/diplodoc-platform/components/assets/87060818/623ec3eb-5f35-4308-9f25-83b23bdc82e7

Screen Reader: https://www.nvaccess.org

**Review by martyanovandrey — COMMENTED** [review]
Hey, the videos are great, it's great that you made them.
There are a couple of things that need to be fixed.

Let's leave the title only for the dc-mini-toc__title and make a div tag -> h2. We will leave the remaining elements as <a> links in the mini-current itself, it will be more correct, this became more noticeable thanks to the video with a screen reader.

Apparently Scrollspy component already adds an aria-current label for mini-current elements, which means we don’t need to put miniTocContent into a separate variable at all and can leave it as is.

File: src/components/MiniToc/MiniToc.tsx [hunk 2]
```diff
@@ -48,8 +48,10 @@ const MiniToc = memo<MinitocProps>(({headings, router, headerHeight, onItemClick
     }
 
     return (
-        <div className={b()}>
-            <div className={b('title')}>{t<string>('title')}:</div>
+        <nav className={b()} aria-label={t('article-navigation')}>
+            <div>
```
Comment (martyanovandrey) [inline src/components/MiniToc/MiniToc.tsx:52]: for what reason you need extra div here ?

**Review by martyanovandrey — APPROVED** [review]

**Pull request merged**